sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -240,7 +239,7 @@ def __init__(self, **kwargs):
     def _from_generated(cls, field, value, read_result):
         return cls(
             value_type=adjust_value_type(value.type) if value else None,
-            label_data=FieldData._from_generated(field, read_result),
+            label_data=None,  # not returned with receipt/supervised
```

Comment:
Didn't we decide to return the name property in the label_data for supervised?

Reply:
Do you mean on `label_data.text`? The original design mapping specifies null for `label_data` with supervised which makes sense since you don't get this info from training with labels.

Reply:
Oh sorry confused this with a Formfield.name. 
>specifies null for label_data

this is correct/